### DISCUSSION TITLE
[Art competition] Program Synthesis

### DISCUSSION THREAD

**Discussion by 702fbtngus** [prosyslab-classroom/cs424-program-reasoning#252] [Art competition]
![DALL·E 2022-11-30 11 57 23 - nervous cook and cooking pot in front of many people, cartoon style](https://user-images.githubusercontent.com/79245586/204697826-eb7d5415-163b-48f6-bf02-6f6d799f4ecc.png)

## Short Description
For me, program synthesis means "nervous cook and cooking pot in front of many people, cartoon style".

## Long Description (up to 3 sentences)
In a cooking contest, the cook should combine various ingredients to make a dish, which should make the gourmets satisfied. It is similar with program synthesis in that I should make use of various grammar to make a program that satisfies each of the constraints.

**Comment by KihongHeo** [comment]
Oh, for you, program synthesis is a stressful task... Sorry!
